### PR TITLE
GH-175 - Updating add issue fix

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
GH-175 - Updating add issue fix

#### Ticket Link
https://github.com/mattermost/mattermost-plugin-todo/issues/175

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,4 +7,5 @@ require (
 	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200804063212-d4dac31b042a
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.6.1
+	golang.org/x/sys v0.0.0-20220823224334-20c2bfdbfe24 // indirect
```
Comment (mickmister) [inline go.mod:10]: This dep update is unrelated to the project so I suggest removing it

File: webapp/src/components/add_issue/add_issue.jsx [hunk 2]
```diff
@@ -77,13 +77,18 @@ export default class AddIssue extends React.PureComponent {
...
             description: '',
         });
+
         if (attachToThread) {
-            submit(message, description, assignee.username, postID);
+            if (assignee) {
+                submit(message, description, assignee.username, postID);
+            } else {
+                submit(message, description, sendTo, postID);
+            }
```
Comment (mickmister) [inline webapp/src/components/add_issue/add_issue.jsx:91]: What's the difference between assigning a todo to someone and sending it to someone?

**Review by mickmister — APPROVED** [review]
LGTM

**Comment by mickmister** [comment]
Looks like the linter is complaining about inconsistencies between `go.mod` and `go.sum`. If you plan on removing the `sys` dependency, then removing the changes in `go.mod` and `go.sum` should fix the issue.

**Comment by asaadmahmood** [comment]
Done @mickmister

File: webapp/src/components/add_issue/add_issue.jsx (hunk 2 above)
Comment (asaadmahmood) [reply webapp/src/components/add_issue/add_issue.jsx:91]: @mickmister I think one assigns it to another person, while the other is just your own.

**Review by larkox — APPROVED** [review]

File: webapp/src/components/add_issue/add_issue.jsx (hunk 2 above)
Comment (mickmister) [reply webapp/src/components/add_issue/add_issue.jsx:91]: Not sure what this means. Sending it to someone means you're sending it to yourself? Maybe there's been a fundamental change to how this works semantically?

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- This issue is resolved now
- Tested with and without assignee
- Tested with and without CRT (as a precaution as this seemed to not change the behavior)
LGTM!

I did find 1 issue but it is not related to this PR. Created https://github.com/mattermost/mattermost-plugin-todo/issues/175 to address it.
